src/optimum/rbln/transformers/models/decoderonly/decoderonly_architecture.py

```diff
@@ -182,7 +182,11 @@ def __init__(
     def get_rotary_emb(self, max_seq_len):
         return RotaryEmbedding(config=self.config, max_seq_len_cached=max_seq_len)
 
-    def convert_to_rbln_causal_lm(self, causal_lm: PreTrainedModel, max_seq_len: int):
+    def convert_to_rbln_causal_lm(
+        self,
+        causal_lm: PreTrainedModel,
+        max_seq_len: int,
+    ):
```

Comment:
```suggestion
    def convert_to_rbln_causal_lm(
        self,
        causal_lm: PreTrainedModel,
        max_seq_len: int,
    ):
```
could you revert this as prev?